src/sage/schemes/elliptic_curves/ell_rational_field.py

```diff
@@ -2268,7 +2280,11 @@ def gens(self, proof=None, **kwds):
         - ``use_database`` -- boolean (default: ``True``); if ``True``, attempts to
           find curve and gens in the (optional) database
 
-        - ``descent_second_limit`` -- (default: 12) used in 2-descent
+        - ``descent_second_limit`` -- (default: 12); logarithmic height bound on
```

Comment:
Also I notice the default here is 12 instead of 8… changing it would be a functional change though.

Reply:
I would leave it. I don't feel comfortable reviewing such a change.